spec/models/zone_spec.rb

```diff
@@ -327,4 +327,11 @@
       expect(message).to be_nil
     end
   end
+
+  context "#valid?" do
+    it "doesn't query for an unchanged record" do
+      alert = FactoryBot.create(:zone)
+      expect { alert.valid? }.not_to make_database_queries
```

Comment:
sorry for the drive by nit, but could you name this something other than an alert please?